playground/dapr/AppHost/Program.cs

```diff
@@ -1,15 +1,21 @@
+using Aspire.Hosting.Dapr;
```

Comment:
We don't want to change the sample, can you write a test instead? We now have dapr tests.

Reply:
cool was not sure. When I started with that PR there were no tests for dapr. :)